### PR TITLE
IDMComputerSolutions,Inc.UEStudio version 22.2.52

### PR DESCRIPTION
**Pull request by Scavy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96842)

### CODE REVIEW

**Comment by Scavy** [comment]
UEStudio is UltraEditStudio by IDM Computer Solutions - Their main domain is ultraedit.com and the download URL domain is downloads.ultraedit.com . They seems to have been sold to Idera Inc. back in 2021 - But is still citing IDM Computer Solutions as the company in the About dialog.

File: manifests/i/IDMComputerSolutions,Inc/UEStudio/22.2.52/IDMComputerSolutions,Inc.UEStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: IDMComputerSolutions,Inc.UEStudio
+PackageVersion: 22.2.52
+PackageLocale: en-US
+Publisher: IDM Computer Solutions, Inc.
+PackageName: UEStudio
```
Comment (stephengillie) [inline manifests/i/IDMComputerSolutions,Inc/UEStudio/22.2.52/IDMComputerSolutions,Inc.UEStudio.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.ultraedit.com/company/about-us
PublisherSupportUrl: https://www.ultraedit.com/support/
PrivacyUrl:   https://www.ideracorp.com/Legal/PrivacyShield
PackageName: UEStudio
PackageUrl: https://www.ultraedit.com/
```

**Comment by Scavy** [comment]
Are there anything wrong with this package? It seems to be stuck somehow. Let me know if there is anything I need to do?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**